Translation correction (#117)
07th-mod/higurashi-console-arcs: Update/some_010.txt

			NULL, "\"Ch-Chisato-chan...", Line_WaitForInput);
	ModPlayVoiceLS(4, 36, "ps3/s04/36/310100879", 256, TRUE);
	OutputLine(NULL, "なっんて、ことをぉ…っ！！」",
			NULL, " What have you done...?!\"", Line_Normal);
	ClearMessage();

//rもはや怒るべきか悲しむべきかわからない感情のまま、ふるふると両手のこぶしを振り上げながら私は、半べその顔で千紗登ちゃんに抗議する。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "もはや怒るべきか悲しむべきかわからない感情のまま、ふるふると両手のこぶしを振り上げながら私は、半べその顔で千紗登ちゃんに抗議する。",
			NULL, "Unsure whether to be angry or sad, I raised both my fists as I objected to Chisato-chan, my face on the verge of tears.", Line_Normal);
	ClearMessage();

//rしかし千紗登ちゃんは『あっはっはっはっ！！』と豪快に笑い飛ばすと、ぽむ！と強く私の肩を叩いてニヤリと目配せした。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "しかし千紗登ちゃんは『あっはっはっはっ！！』と豪快に笑い飛ばすと、ぽむ！と強く私の肩を叩いてニヤリと目配せした。",

> 			NULL, "But Chisato-chan just laughed hearty, then she gave me a strong pat on the shoulder with a huge grin.", Line_Normal);

Reviewer: ```suggestion
			NULL, "But Chisato-chan just gave me a hearty laugh, then a strong pat on the shoulder with a huge grin on her face.", Line_Normal);
```

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#fda762>千紗登</color>", NULL, "<color=#fda762>Chisato</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 37, "ps3/s04/37/310300563", 256, TRUE);
	OutputLine(NULL, "「というわけで、皆の衆！",

> 			NULL, "\"So there you people have it!", Line_WaitForInput);

Reviewer: ```suggestion
			NULL, "\"So there you have it, everyone!", Line_WaitForInput);
```

//r祝ってくれているのか、それとも嫉妬されているのかよくわからないほど、男子プラス女子たちの祝福は手荒で容赦がなかった。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "祝ってくれているのか、それとも嫉妬されているのかよくわからないほど、男子プラス女子たちの祝福は手荒で容赦がなかった。",

> 			NULL, "It was unclear if the boys and girls were really celebrating. Their congratulations were so rough and unforgiving that it made him wonder if they were instead jealous.", Line_Normal);

Reviewer: ```suggestion
			NULL, "It was unclear if the boys and girls were really celebrating. Their congratulations were so rough and unforgiving that it made him wonder if they were actually jealous.", Line_Normal);
```

//r荷物が置きっ放しだったが、仕方がない。このまま直接美術室に、…そう思いながら足を踏み出しかけて、暁はふと思い出す。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "荷物が置きっ放しだったが、仕方がない。このまま直接美術室に、…そう思いながら足を踏み出しかけて、暁はふと思い出す。",
			NULL, "He left his bag behind, but it was too late to do anything about that. He began heading in the direction of the art room... but a thought suddenly came to mind after he took the first step.", Line_Normal);
	ClearMessage();

//r…確か、クラスメイトの噂好きが美術部員だった。今頃はこの話で盛り上がっているはずだから、美術室でもさっきの二の舞が繰り広げられることだろう。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "…確か、クラスメイトの噂好きが美術部員だった。今頃はこの話で盛り上がっているはずだから、美術室でもさっきの二の舞が繰り広げられることだろう。",

> 			NULL, "...His classmates in the art club were fond of rumors. They were likely already up in arms about this laes revelation, so going to the art room at this point in time was essentially a death sentence.", Line_Normal);

Reviewer: ```suggestion
			NULL, "...His classmates in the art club were fond of rumors. They were likely already up in arms about this latest revelation, so going to the art room at this point in time was essentially a death sentence.", Line_Normal);
```
